go/vt/sqlparser/sql.y

```diff
@@ -1931,6 +1934,12 @@ char_type:
   {
     $$ = ColumnType{Type: string($1), Length: $2, Charset: $3}
   }
+| CHAR length_opt BYTE
+  {
+    // CHAR BYTE is an alias for binary. See also:
+    // https://dev.mysql.com/doc/refman/8.0/en/string-type-syntax.html
```

Comment:
This is a fun one, this is actually an alias for `binary`. See also the linked page, under:

```
The [CHAR BYTE](https://dev.mysql.com/doc/refman/8.0/en/binary-varbinary.html) data type is an alias for the [BINARY](https://dev.mysql.com/doc/refman/8.0/en/binary-varbinary.html) data type. This is a compatibility feature.
```

Reply:
gah